src/platform/common/utils/localize.ts

```diff
@@ -244,27 +271,36 @@ export namespace DataScience {
         'Failed to connect to password protected server. Check that password is correct.'
     );
     export const rawKernelProcessNotStarted = localize(
-        'DataScience.rawKernelProcessNotStarted',
+        { key: 'DataScience.rawKernelProcessNotStarted', comment: ['{Locked="kernel"}'] },
         'Raw kernel process was not able to start.'
     );
     export const rawKernelProcessExitBeforeConnect = localize(
-        'DataScience.rawKernelProcessExitBeforeConnect',
+        { key: 'DataScience.rawKernelProcessExitBeforeConnect', comment: ['{Locked="kernel"}'] },
         'Raw kernel process exited before connecting.'
     );
     export const unknownMimeTypeFormat = localize(
-        'DataScience.unknownMimeTypeFormat',
+        { key: 'DataScience.unknownMimeTypeFormat', comment: ['{Locked="Mime type"}'] },
         'Mime type {0} is not currently supported'
     );
-    export const exportDialogTitle = localize('DataScience.exportDialogTitle', 'Export to Jupyter Notebook');
-    export const exportDialogFilter = localize('DataScience.exportDialogFilter', 'Jupyter Notebooks');
+    export const exportDialogTitle = localize(
+        { key: 'DataScience.exportDialogTitle', comment: ['{Locked="Notebook"}'] },
+        'Export to Jupyter Notebook'
+    );
+    export const exportDialogFilter = localize(
+        { key: 'DataScience.exportDialogFilter', comment: ['{Locked="Notebooks"}'] },
+        'Jupyter Notebooks'
+    );
     export const exportDialogComplete = localize('DataScience.exportDialogComplete', 'Notebook written to {0}');
     export const exportDialogFailed = localize('DataScience.exportDialogFailed', 'Failed to export notebook. {0}');
     export const exportOpenQuestion1 = localize('DataScience.exportOpenQuestion1', 'Open in editor');
     export const runCellLensCommandTitle = localize('jupyter.command.jupyter.runcell.title', 'Run Cell');
-    export const importDialogTitle = localize('DataScience.importDialogTitle', 'Import Jupyter Notebook');
-    export const importDialogFilter = localize('DataScience.importDialogFilter', 'Jupyter Notebooks');
+    export const importDialogTitle = localize(
+        { key: 'DataScience.importDialogTitle', comment: ['{Locked="Notebook"}'] },
+        'Import Jupyter Notebook'
+    );
+    export const importDialogFilter = () => 'Jupyter Notebooks';
```

Comment:
I'm assuming this would never change.

Reply:
If `Jupyter` cannot change here, then it should be locked everywhere else.